### PR TITLE
Fix maven build on JDK 8 / 9+

### PR DESCRIPTION
**Pull request by bourgesl**
Please review this simple pull request to fix maven build:
- import few (fake) apple classes needed by src (base)
- use multiple source folders (src, src-jdk8) and src-jdk9 for extra Java 9+ classes
- use antrun to compile src-jdk9 (JDK9) using needed --add-exports args
- package a multi-release jar file (8/9)
- code formatting

Please adjust the JDK9 path in the pom.xml:
`<java9.jdk>/home/bourgesl/apps/jdk-9/</java9.jdk>`

### CODE REVIEW

**Comment by bourgesl** [comment]
Please adjust the JDK9 path in the pom.xml:
<java9.jdk>/home/bourgesl/apps/jdk-9/</java9.jdk>

I does not work as it is.

**Comment by jidesoft** [comment]
What's a common way to specify jdk path in maven?

**Comment by bourgesl** [comment]
Maven recommends using maven-toolchains-plugin, I will try using it:
http://maven.apache.org/guides/mini/guide-using-toolchains.html

**Comment by tferr** [comment]
@bourgesl, @jidesoft, Just wondering: do you have any updates on this?

**Comment by bourgesl** [comment]
It works for me, just do it

File: pom.xml [hunk 1]
```diff
@@ -32,15 +35,27 @@
...
         <developer>
             <id>jidesoft</id>
             <name>jidesoft</name>
             <email>support@jidesoft.com</email>
         </developer>
     </developers>
+
+    <properties>
+        <!-- path to JDK9+ -->
+        <java9.jdk>/home/bourgesl/apps/jdk-9/</java9.jdk>
```
Comment (apjanke) [inline pom.xml:49]: This `/home/bourgesl` path doesn't seem portable?

**Pull request merged**